app/misc/git.ts

```diff
@@ -1773,6 +1773,7 @@ function editUpstream() {
       .then(function (repo) {
       repository = repo;
       Git.Remote.delete(repository, 'upstream').then(function(result) {
+        addCommand("git remote add upstream " + upstreamRepoPath);
```

Comment:
I think it should do a delete first -- then the add?

Reply:
good catch, it's updated now